src/renderer/services/binance/transaction.ts

```diff
@@ -33,8 +33,8 @@ const tx$ = ({
     switchMap((r) => (O.isSome(r) ? Rx.of(r.value) : Rx.EMPTY)),
     switchMap((client) =>
       memo
-        ? Rx.from(client.vaultTx(to, amount.amount().toNumber(), symbol, memo))
-        : Rx.from(client.normalTx(to, amount.amount().toNumber(), symbol))
+        ? Rx.from(client.vaultTx({ addressTo: to, amount: amount.amount().toNumber(), asset: symbol, memo }))
```

Comment:
just noticed that we use here number value for amount. but actually we have BigNumber and with such casting we have loose percision of user's input. its not related to PR's changes just a thoughts. What do u think ?

Reply:
Good point :+1:  Will change it after merging https://gitlab.com/thorchain/asgardex-common/asgardex-binance/-/merge_requests/17